.travis/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# OS-specific dependencies installer for Travis CI, which is run during
+# the `install` step.
+# Copyright (C) 2015-2019 IPONWEB Ltd. See Copyright Notice in COPYRIGHT
```

Comment:
Decided not to change the year - I think it's better to change all such comments across the code base in one commit.